src/azure-cli/azure/cli/command_modules/appservice/_validators.py

```diff
@@ -131,6 +131,24 @@ def validate_functionapp_on_containerapp_site_config_set(cmd, namespace):
             "Please use the following command instead: az functionapp config container set")
 
 
+def validate_functionapp_on_containerapp_container_settings_delete(cmd, namespace):
+    resource_group_name = namespace.resource_group_name
+    name = namespace.name
+    if is_centauri_functionapp(cmd, resource_group_name, name):
+        raise ValidationError(
+            "Invalid command. This is not supported for Azure Functions on Azure Container app environments.",
```

Comment:
Can we say this is not support _in current stage_ or something similar, so we don't leave cx an impression this command will never work as PM may ask to have it support in future.